__tests__/components/SignerBaseFilter.spec.ts

```diff
@@ -0,0 +1,165 @@
+            {
+                OptionGroup: true,
+                Icon: true,
+            },
+        );
+
+    describe('component', () => {
+        const runBasicSelectedSignerInMountTests = (props, expectResult) => {
+            // Arrange:
+            const wrapper = getSignerBaseFilterWrapper({
+                rootSigner: mockSigner,
+                ...props,
+            });
+
+            const vm = wrapper.vm as SignerBaseFilterTs;
+
+            // Act + Assert:
+            expect(vm.selectedSigner).toBe(expectResult);
+        };
+
+        test('renders only account in the list when parent signer does not exist', () => {
+            // Arrange:
+            const wrapper = getSignerBaseFilterWrapper({
+                rootSigner: {
+                    label: 'wallet 1',
+                    address: simpleWallet1.address,
+                    multisig: false,
+                    requiredCosigApproval: 0,
+                    requiredCosigRemoval: 0,
+                },
+            });
+
+            // Act + Assert:
+            const options = wrapper.findAll('option');
+            expect(options.length).toBe(1);
+            expect(options.at(0).text()).toBe('wallet 1');
+            expect(options.at(0).attributes().value).toBe(simpleWallet1.address.plain());
+        });
+
+        test('renders account and multisig address in the list when parent signer exists', () => {
+            // Arrange:
+            const wrapper = getSignerBaseFilterWrapper({
+                rootSigner: {
+                    ...mockSigner,
+                    parentSigners: [
+                        {
+                            ...mockSigner,
+                            label: 'wallet 2',
+                            address: simpleWallet2.address,
+                            multisig: true,
+                        },
+                    ],
+                },
+            });
+
+            // Act + Assert:
+            const options = wrapper.findAll('option');
+            expect(options.length).toBe(2);
+            expect(options.at(0).text()).toBe('wallet 1');
+            expect(options.at(0).attributes().value).toBe(simpleWallet1.address.plain());
+            expect(options.at(1).text()).toBe('(Multisig)wallet 2');
+            expect(options.at(1).attributes().value).toBe(simpleWallet2.address.plain());
+        });
+
+        test('created set selected signer to chosen signer when isAggregate is true and chosen signer does not empty', () => {
+            const chosenSigner = simpleWallet2.address.plain();
+            runBasicSelectedSignerInMountTests({ isAggregate: true, chosenSigner }, chosenSigner);
+        });
+
+        test('created set selected signer to current signer when isAggregate is false and chosen signer is empty', () => {
+            const chosenSigner = '';
+            runBasicSelectedSignerInMountTests({ isAggregate: false, chosenSigner }, simpleWallet1.address.plain());
+        });
```

Comment:
should you test the two other combinations too for completeness?

Reply:
agree, I will update 👍🏼